api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -35,14 +35,16 @@ message DisperseBlobRequest {
   // significant bits. The integer must stay in the valid range to be interpreted as a field element on the bn254 curve.
   // The valid range is 0 <= x < 21888242871839275222246405745257275088548364400416034343698204186575808495617.
   // If any one of the 32 bytes elements is outside the range, the whole request is deemed as invalid, and rejected.
-  bytes data = 1;
+  bytes blob = 1;
   // The header contains metadata about the blob.
   //
   // This header can be thought of as an "eigenDA tx", in that it plays a purpose similar to an eth_tx to disperse a
   // 4844 blob. Note that a call to DisperseBlob requires the blob and the blobHeader, which is similar to how
   // dispersing a blob to ethereum requires sending a tx whose data contains the hash of the kzg commit of the blob,
   // which is dispersed separately.
   common.v2.BlobHeader blob_header = 2;
+  // signature over keccak hash of the blob_header that can be verified by blob_header.payment_header.account_id
```

Comment:
I'm fussy about this but I think it would be good to be super precise about the exact thing we are signing over. "keccak hash of blob_header" is not precise as it doesn't describe the serialization procedure (nor the fact that we use multi-level hashing).
<img width="679" alt="image" src="https://github.com/user-attachments/assets/85cb6dc4-5ba3-4904-a223-625dd935ca1e" />

Perhaps this is one reason to keep `BlobKey` around, and be super precise in its documentation that it refers to a very specific hashing/serialization combination of the BlobHeader? Then we can just use `BlobKey` everywhere and refer people to its documentation comment.